Headers/IOKit/80211/IO80211Controller.h

```diff
@@ -127,255 +127,197 @@ struct apple80211_lteCoex_report;
 typedef IOReturn (*IOCTL_FUNC)(IO80211Controller*, IO80211Interface*, IO80211VirtualInterface*, apple80211req*, bool);
 extern IOCTL_FUNC gGetHandlerTable[];
 extern IOCTL_FUNC gSetHandlerTable[];
+#define __int64 int
```

Comment:
These are hacks from IDA, better to replace them with C types to be honest.

Reply:
ok.